### ISSUE TITLE
Dimensions are wonky

### ISSUE THREAD

**Issue by workergnome** [american-art/SAAM#9]
For http://data.americanartcollaborative.org/page/saam/object/id/10086, the dimensions are strange.  Can you take a look?  There's a single dimension, which is both width & height, and is also both cm and inches.

**Comment by rkbrasse** [comment]
Yes that is how they were stored in the data.  If consistent I'm sure they could be parsed into more appropriate representation.

**Comment by workergnome** [comment]
@bsnikhila— can you look and see if there's a way to use this data?  Right now, it's not very meaningful.

**Comment by bsnikhila** [comment]
There are 2 sources, WebObjDimensionsSplit_view and the dimensions in the WebObjCaption file. Currently only the data in the WebObjCaption file has been mapped. Should I map the data in the WebObjDimensionsSplit_view along with dimensions in WebObjCaption, or only dimensions in WebObjDimensionsSplit_view?
